### PR TITLE
[IMP] Welcome auth_signup_approval

### PR DESCRIPTION
**Pull request by maitaoriana**

### CODE REVIEW

File: auth_signup_approval/views/mail_template.xml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+            <field name="name">Auth Signup: User pending approval</field>
+            <field name="model_id" ref="base.model_res_users"/>
+            <field name="email_from"><![CDATA["${object.company_id.name|safe}" <${(object.company_id.email or user.email)|safe}>]]></field>
+            <!--<field name="email_to">${object.email|safe}</field>-->
+            <field name="subject"><![CDATA[New Auth Signup: ${object.name}]]></field>
+            <field name="body_html"><![CDATA[
+<div style="padding:0px;width:600px;margin:auto;background: #FFFFFF repeat top /100%;color:#777777">
+    <p>Dear,</p>
+    <p>
+        The user ${object.name} has requested to register on the site, is awaiting your approval. <br/>
```
Comment (atpollmann) [inline auth_signup_approval/views/mail_template.xml]: ```The user ${object.name} has requested to sign up on the site. The request is awaiting your approval. <br/>```

File: auth_signup_approval/views/mail_template.xml [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+            <td valign="center" width="200" style="padding:10px 10px 10px 5px;font-size: 12px">
+                <img src="/logo.png" style="padding: 0px; margin: 0px; height: auto; width: 80px;" alt="${user.company_id.name}" />
+            </td>
+        </tr></tbody>
+    </table>
+</div>
+<div style="padding:0px;width:600px;margin:auto;background: #FFFFFF repeat top /100%;color:#777777">
+    <p>Dear ${object.name},</p>
+    <p>
+    We regret to notify you that your request to register on our website has been rejected,
```
Comment (atpollmann) [inline auth_signup_approval/views/mail_template.xml]: ```We're sorry, but your request to register in our site has been rejected.```

File: auth_signup_approval/views/mail_template.xml [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+    if you have any questions or comments, do not hesitate to contact us.
```
Comment (atpollmann) [inline auth_signup_approval/views/mail_template.xml]: ```For any inquiry about the registration approval process, please <a href='<form_url>'>contact us</a>.```

File: auth_signup_approval/controllers/main.py [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+        # Remove password
+        values["password"] = ""
+
+        message = self.create_user(values, qcontext.get("token"), login)
+        if message:
+            qcontext["error"] = message
+            return request.render("auth_signup.signup", qcontext)
+
+        qcontext["message"] = _(
+            "Thank you for you singup we will sent you activation email "
```
Comment (maitaoriana) [inline auth_signup_approval/controllers/main.py]: @atpollmann

Comment (atpollmann) [reply auth_signup_approval/controllers/main.py]: Thank you for signing up in our site.
Your information it's being reviewed by one of our administrators.
If your account is approved, you will receive an email with instructions on how to activate it.
<link to return to homepage>

Gracias por registrarse en nuestro sitio.
Su información esta siendo revisada por uno de nuestros administradores.
Si su cuenta es aprovada, recibirá un correo con instrucciones de cómo activarla.
<link to return to homepage>

**Pull request merged**